asmcomp/x86_proc.ml

```diff
@@ -50,6 +53,9 @@ let system = match Config.system with
   | "mingw64" -> S_mingw64
   | "win64" -> S_win64
   | "linux" -> S_linux
+  | "freebsd" -> S_freebsd
+  | "netbsd" -> S_netbsd
+  | "openbsd" -> S_openbsd
```

Comment:
Is it worth separating between these? Or perhaps a `S_bsd` constructor would be better?

Reply:
They are entirely different operating systems these days, with decades of divergence and differing toolchains. Keeping them separate (as the configure script also does) would be better I think.